web/groups.go

```diff
@@ -119,31 +117,19 @@ func (s *QuickFeedService) updateGroup(ctx context.Context, sc scm.SCM, request
 		return fmt.Errorf("failed to get %s repository for group %q: %w", course.Code, group.Name, err)
 	}
 	if repo == nil {
-		// no group repository exists; create new repository for group
-		if request.Name != "" && newGroup.ScmTeamID < 1 {
```

Comment:
Should we still check for `request.Name != ""` and update `newGroup.Name` as before? 

We will get here only if `repo == nil` meaning that there is no repo recorded in the database, and thus we haven't created it yet. Thus, we can still change the name according to the `request`... Or perhaps I'm missing something?

Reply:
I don't think it's necessary as a similar check is being done in `s.newGroup`.